src/configuration/commandline_variables.md

```diff
@@ -0,0 +1,50 @@
+## Command-line Variables & Scripts
+
+This section provides guidance on retrieving command-line environment variables and incorporating scripts, such as `rizinrc`, within the Rizin environment.
+
+### Command-line Variables
```

Comment:
Something doesn't seem right. What kind of variables are these exactly.  

"command-line environment variables" by itself is a bit weird. Environment variables work and are relevant regardless of whether you are working with command-line. They can be used to configure GUI programs as much as commandline tools, and windows even has a GUI for setting them up.

If they really are environment variables, are they environment variables that rizin uses for configuration  or are they variables that rizin sets for child process. Changing a lot of them  like RZ_VERSION from commandline doesn't make sense.   And runing `!env` inside rizin shell suggests that this isn't the case either. Rizin does setup a couple of  variables for child processes but that's a completely different set. Thinking about it more I guess these are more of a build configuration variables - nothing to do with command line or environment variables.

Reply:
i tried to find a good name for this sections and is hard. :/